src/view/components/microbit/MicrobitImage.tsx

```diff
@@ -69,6 +69,7 @@ const setupButton = (
     buttonElement.onmouseleave = e => {
         eventTriggers.onMouseLeave(e, key);
     };
+    console.log("buttons should be enabled");

```

Comment:
Was this log on purpose? One way to avoid getting cluttering log messages is to use `console.debug` instead (although it doesn't seem like it makes such a difference on typescript, other than the color 😟) and adding some context brackets, e.g. "[Input] Buttons should have been enabled."

Reply:
i'm removing this one :), seems like debug might be useful since it's hidden from regular users